docs/report/001.ipynb

```diff
@@ -46,7 +46,7 @@
    },
    "outputs": [],
    "source": [
-    "%pip install -q black==21.5b2 jax==0.4.28 jaxlib==0.4.28 matplotlib==3.4.2 numpy==1.22 sympy==1.8"
```

Comment:
Note on `pip`: `numpy==1.22` installs `numpy==1.22.0`, not the latest patch of v1.22 (in this case `numpy==1.22.4`). If you want to find the latest version of that patch, use either
```shell
pip install -U 'numpy==1.22.*'
```
or
```shell
pip install -U 'numpy~=1.22'
```
As you noted (https://github.com/ComPWA/compwa.github.io/issues/272#issuecomment-2104557801), you can find more info about that here https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts.

In this commit, I upgraded even further to the latest version of NumPy v1 because I got some dependency conflict during runtime (!) with matplotlib.

Reply:
Good that you notice!